apps/src/code-studio/components/header/ProjectShare.jsx

```diff
@@ -2,10 +2,22 @@ import React from 'react';
 import i18n from '@cdo/locale';
 import {shareProject} from '../../headerShare';
 import {styles} from './EditableProjectName';
+import LabRegistry from '@cdo/apps/labs/LabRegistry';
+import {shareProjectV2} from '@cdo/apps/labs/header/headerShareV2';
 
 export default class ProjectShare extends React.Component {
   shareProject = () => {
-    shareProject(dashboard.project.getShareUrl());
+    if (LabRegistry.getInstance().getProjectManager() !== null) {
```

Comment:
Would it be helpful to wrap this check in a single function call (maybe something like `LabsV2.isEnabled()` or whatever we end up going with) so it can be reused and the naming is clearer?

Reply:
I added a function to `LabRegistry` called `hasEnabledProjects`. I think it may be worth renaming `LabRegistry` to `Lab2Registry`, but I'll do that as a follow-up.